cron-parameters/README.md

```diff
@@ -0,0 +1,112 @@
+<!--
```

Comment:
This readme probably shouldn't include everything that was in `simple-cron` - it should probably just link to it and include new information.